src/server/server.test.ts

```diff
@@ -61,14 +61,14 @@ describe('server test suite', () => {
     await server.start();
     const port = server.http.address().port;
     await request(`http://localhost:${port}`).get('/').expect('HANDLER_B').catch(fail);
-    server.stop();
+    return server.stop();
   });
 
   it('should fail to start an already started server', async () => {
     const server = new Server(express());
     await server.start();
     await expect(server.start()).rejects.toThrow(/already started/);
-    server.stop();
+    return server.stop();
   });
```

Comment:
I knew I missed few places in the tests... Shoutout for finding and fixing these! 👏